### PR TITLE
Scale error in mesh optimizer so it uses absolute scale.

### PR DESCRIPTION
**Pull request by fire**
Test procedure.

1. Download a model.
2. Go to the wireframe mode. Go close and zoom out.
3. Note where the lod transitions are.
4. Switch to the normal mode. Go close and zoom out.
5. See if there's a noticable difference.

Currently the normals get flipped, but these numbers are tuned on https://kitbash3d.com/collections/kits/products/utopia, bistro and the subd list.

### CODE REVIEW

**Comment by fire** [comment]
Switch to simplify sloppy for another try.

Update to meshoptimizer e3f53f66e7a35b9b8764bee478589d79e34fa698.

**Comment by fire** [comment]
I think master is broken.

**Review by reduz — APPROVED** [review]

**Comment by fire** [comment]
Modified so that the sloppy method is disabled. Be more conservative about breaking people's meshes.

File: editor/import/scene_importer_mesh.cpp [hunk 1]
```diff
@@ -157,20 +163,52 @@ void EditorSceneImporterMesh::generate_lods() {
...
+		float abs_target_error = target_error / mesh_scale;
 		while (index_target > min_indices) {
 			float error;
 			Vector<int> new_indices;
 			new_indices.resize(indices.size());
-			size_t new_len = SurfaceTool::simplify_func((unsigned int *)new_indices.ptrw(), (const unsigned int *)indices.ptr(), indices.size(), (const float *)vertices_ptr, vertex_count, sizeof(Vector3), index_target, 1e20, &error);
-			print_line("shoot for " + itos(index_target) + ", got " + itos(new_len) + " distance " + rtos(error));
+			size_t new_len = SurfaceTool::simplify_func((unsigned int *)new_indices.ptrw(), (const unsigned int *)indices.ptr(), indices.size(), (const float *)vertices_ptr, vertex_count, sizeof(Vector3), index_target, abs_target_error, &error);
 			if ((int)new_len > (index_target * 120 / 100)) {
+				// Attribute discontinities break normals.
```
Comment (akien-mga) [inline editor/import/scene_importer_mesh.cpp:176]:
```suggestion
				// Attribute discontinuities break normals.
```

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**